tests/main/confinement-classic/task.yaml

```diff
@@ -1,10 +1,13 @@
 summary: trivial snap with classic confinement runs correctly
+
+systems: [-ubuntu-core-16-*, -fedora-*, -opensuse-*]
```

Comment:
This test should work OK on openSUSE. In general please indicate why each system is disabled. Sometimes this is deliberate for a reason (which should be written down) but often it is just a temporary workaround because something else is not in place. By placing a comment it is easier to audit and review this later.